docs/src/FAQ.md

```diff
@@ -76,3 +76,17 @@ ChainRulesTestUtils.jl has some dependencies, so it is a separate package from C
 This means your package can depend on the light-weight ChainRulesCore.jl, and make ChainRulesTestUtils.jl a test-only dependency.
 
 Remember to read the section on [On writing good `rrule` / `frule` methods](@ref).
+
+## How do chain rules work for complex functions?
+
+`ChainRules.jl` follows the convention that `frule` applied to a function `f(x + im*y) = u(x,y) + im*v(x,y)` with perturbation `ẋ + im*ẏ` returns
+```
+∂u/∂x * ẋ + ∂u/∂y * ẏ + im * ( ∂v/∂x * ẋ    ∂v/∂y * ẏ )
+```
+and similarly `rrule` applied to the same function with adjoint `ū + im*v̄` returns
```

Comment:
```suggestion
and similarly `rrule` applied to the same function evaluates the function and return a pullback which, when applied to adjoint `ū + im*v̄`, returns
```

Reply:
I get that my version wasn't 100% accurate, but I think it is justified to be slightly imprecise here for the sake of getting straight to the point. 

If we do make this change than we should also say that `frule` returns the function value and the derivative (cf. first half of sentence).

Reply:
FWIW, i prefer accuracy over brevity. I agree it'd be good to be consistent in the level of detail for both `frule` and `rrule`

Reply:
Most people seem to agree with you, so I guess that's what we're sticking with.